### PR TITLE
Rdm 11544

### PR DESCRIPTION
**Pull request by Thor-tech-of-metal**
RDM-11544: Unspec elastic search is giving 400
               [RDM-11544] (https://tools.hmcts.net/jira/browse/RDM-11544).

### CODE REVIEW

File: elastic-search-support/src/main/resources/application.yml [hunk 1]
```diff
@@ -16,7 +16,7 @@ elasticsearch:
     defaultBoolean: '{"type" : "boolean"}'
     defaultLong: '{"type" : "long"}'
     defaultDouble: '{"type" : "double"}'
-    ccdPhoneUK: '{"type" : "text", "analyzer": "phone_number_analyzer"}'
+    ccdPhoneUK: '{"type" : "text", "fields": { "keyword": { "type": "keyword", "ignore_above": 256, "normalizer": "lowercase_normalizer"}}, "analyzer": "phone_number_analyzer"}'
     ccdDocument: '{
                     "properties":{
```
Comment (Thor-tech-of-metal) [inline elastic-search-support/src/main/resources/application.yml:21]: ccdPhoneUK has been also fixed in this ticket

Comment (danlysiak) [reply elastic-search-support/src/main/resources/application.yml:21]: Thanks 👍

**Review by danlysiak — APPROVED** [review]
👍

**Review by Olawale — APPROVED** [review]
LGTM

**Pull request merged**